go.mod

```diff
@@ -9,12 +9,8 @@ replace (
 	github.com/vmware-tanzu/tanzu-framework/capabilities/client => ./capabilities/client
 	github.com/vmware-tanzu/tanzu-framework/cli/core => ./cli/core
 	github.com/vmware-tanzu/tanzu-framework/cli/runtime => ./cli/runtime
```

Comment:
Will these remain as a temporary thing?

Reply:
Yes, these are temporarily kept because we've to conclude on the cli deliverable of `buildinfo.go`. I'll remove these references with cli directory removal.

Reply:
Updated in latest commit